packages/appstore/src/components/onboarding-new/static-applauncher.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Icon, Button, Text } from '@deriv/components';
+import { formatMoney } from '@deriv/shared';
+import { Localize, localize } from '@deriv/translations';
+
+import './static-applauncher.scss';
+
+type TStaticAppLauncher = {
+    icon_type: 'USD' | 'EUR' | 'Bitcoin' | 'Ethereum' | 'Litecoin';
+    is_item_blurry?: boolean;
+    is_grey?: boolean;
+    is_eu_user: boolean;
+};
+
+const eu_currency_config = {
+    EUR: 'Euro',
+};
+
+const eu_icon_config = {
+    EUR: 'EUR',
+};
+
+const crypto_icon_config = {
+    USD: 'USD',
+    Bitcoin: 'BTC',
+    Ethereum: 'ETH',
+    Litecoin: 'LTC',
+};
+
+const crypto_config = {
+    USD: 'US Dollar',
+    Bitcoin: 'Bitcoin',
+    Ethereum: 'Ethereum',
+    Litecoin: 'Litecoin',
+};
+
+type TCryptoIcon = {
+    currency: string;
+    is_eu_user: boolean;
+};
+
+type TCryptoConfig = {
+    currency: string;
+    is_item_blurry?: boolean;
+    is_eu_user: boolean;
+};
+
+const CryptoIcon = ({ currency, is_eu_user }: TCryptoIcon) => {
+    const currency_icon_classname =
+        currency === ('USD' && 'EUR') ? 'static-applauncher__icon--selected' : 'static-applauncher__icon--blurry';
```

Comment:
it's better not to hardcode these currencies and define a object for them

Reply:
There are just 2 values which are used here and they are static.
Using an object for this matter is just adding more lines of code because these are just two abbreviation with 3 words.
It does not seem as an improvement IMO. What do you think ? @mahdiyeh-fs @matin-binary